iFixit/react-commerce
# Add shipping restriction details hover

closes #729 

### QA

1. Visit Vercel preview
2. You can check a product like [iopener](https://react-commerce-git-add-shipping-restriction-detai-edc61e-ifixit.vercel.app/products/iopener) that has been modified in test to provide both a single and a double restriction 
3. As stated in the document, only the first restriction in the array should be shown. 
4. `fulfilled_via_dropshipping` is the only exception to the rule above: if present it needs always to be shown
5. Check that is working also on mobile
